tests/scripts/all.sh

```diff
@@ -1964,6 +1965,12 @@ component_test_no_use_psa_crypto_full_cmake_asan() {
 component_test_psa_crypto_config_accel_ecdsa () {
     msg "test: MBEDTLS_PSA_CRYPTO_CONFIG with accelerated ECDSA"
 
+    # Algorithms and key types to accelerate
+    loc_accel_list="ALG_ECDSA ALG_DETERMINISTIC_ECDSA KEY_TYPE_ECC_KEY_PAIR KEY_TYPE_ECC_PUBLIC_KEY"
+
+    # Configure and build the test driver library
```

Comment:
```suggestion
    # Configure and build the PSA accelerated test driver library
```
Just to differentiate it from the 2nd build below...

Reply:
Ah, I can't believe I used the same comment in both places while I clearly meant to differentiate them. Thanks for catching that.